app/src/features/rules/components/RulesList/components/RulesTable/hooks/useRuleTableActions.tsx

```diff
@@ -68,9 +68,11 @@ const useRuleTableActions = () => {
       const isRecordRule = isRule(record);
 
       //Push Notify
-      newStatus === RecordStatus.ACTIVE
-        ? toast.success(`${isRecordRule ? "Rule" : "Group"} is now ${newStatus.toLowerCase()}`)
-        : toast.success(`${isRecordRule ? "Rule" : "Group"} is now ${newStatus.toLowerCase()}`);
+      if (showToast) {
```

Comment:
When is showToast optional?

Reply:
For free plan users ,to toggle the status to off of premium rules when table mounts.